test/utils/wait_until.cxx

```diff
@@ -56,4 +56,20 @@ wait_until_collection_manifest_propagated(std::shared_ptr<couchbase::core::clust
     }
     return propagated;
 }
+
+bool
+wait_until_user_present(std::shared_ptr<couchbase::core::cluster> cluster, const std::string& username)
+{
+    auto present = test::utils::wait_until([cluster, username]() {
+        couchbase::core::operations::management::user_get_request req{};
+        req.username = username;
+        auto resp = test::utils::execute(cluster, req);
+        return resp.user.username >= username;
+    });
+    if (present) {
+        // FIXME: The above check does not wait for all nodes to be up to date
```

Comment:
True - this is always an issue with the management calls.   Any calls after this that assume the cluster completely agrees could fail.   The above call to `wait_until_cluster_manifest_propogated`, same thing.  We can live with this, no prob.

Reply:
But -- why the >= username?  Does the user_get_response just have an error (like `couchbase::errc::management::user_not_found`)?   I didn't look, but I'm fairly sure it should.